<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createRoomForm</class>
 <widget class="QWidget" name="createRoomForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pb_cancel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_commit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>280</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>创建</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_mode">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>游戏模式</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_audio">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>61</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>语音</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_video">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>61</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>视频</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_playMethod">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>游戏玩法</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmb_playMethod">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>基础玩法</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="lb_playerNum">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>玩家人数</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sb_playerNum">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_level">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>等级限制</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmb_level">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>无限制</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10级以上</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20级以上</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>30级</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="lb_passwd">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>190</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>设置房间密码</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_passwd">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>21</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_ispasswd">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_passwd">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
